applications/fov-quicklook/templates/coordinator.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: quicklook-coordinator
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      app: quicklook-coordinator
+  template:
+    metadata:
+      labels:
+        app: quicklook-coordinator
+    spec:
+      containers:
+        - name: quicklook-coordinator
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command:
+            - sh
+            - -c
+            - |
+              set -e
+              alembic upgrade head
+              exec python -m quicklook.coordinator.api
+          env:
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: quicklook-db
+                  key: password
+            - name: QUICKLOOK_db_url
+              value: postgresql://quicklook:$(DB_PASSWORD)@quicklook-db:5432/quicklook
+            {{- include "fov-quicklook.env.s3_tile" . | nindent 12 }}
+            {{- include "fov-quicklook.env.s3_repository" . | nindent 12 }}
+          ports:
+            - containerPort: 9501
+          securityContext:
```

Comment:
You should provide the ability to set additional information like resource requests and limits in these deployments.  Another thing that may be necessary to configure, especially for the generator, is `tolerations`.

Reply:
fixed in ab677313707ad170ca8132d891e607a576a5bb28, c2f52033c43db022fd6104a2e5d5897db4e29c9f, 192b0f87215b0582d166c0d215506bad70ce8618